spec/factories/file_attachment.rb

```diff
@@ -0,0 +1,20 @@
+require './lib/pivotal-tracker/file_attachment'
+
+FactoryGirl.define do
+  factory :file_attachment, class: PivotalTracker::FileAttachment do
+    sequence(:id)
+    filename      'filename'
+    created_at    DateTime.now
+    uploader_id   1
+    thumbnailable true
+    height        1
+    width         1
+    size          1
+    download_url  { Faker::Internet.url }
+    content_type  'content_type'
+    uploaded      true
+    big_url       { Faker::Internet.url }
+    thumbnail_url { Faker::Internet.url }
+    kind          'file_attachment'
+  end
```

Comment:
@lucyconklin I interpret that as a string that declares it's type. AKA I support your move.

<img width="637" alt="screen shot 2017-02-03 at 10 47 35 am" src="https://cloud.githubusercontent.com/assets/19894032/22601879/320fc97e-e9fe-11e6-9119-2252a494880c.png">